src/hash_table.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import annotations
+from typing import Union
+import random
+
+
+class Node:
+    def __init__(self, key: str, val: Union[str, float], next: Node = None):
+        self.key = key
+        self.val = val
+        self.next = next
+
+
+class ChainHashTable:
+    def __init__(self, n: int):
+        self.__cap = n
+        self.__size = 0
+        self.__bucket = []
+        for _ in range(n):
+            self.__bucket.append(Node(None, None))
+
+    def cap(self) -> int:
+        return self.__cap
+
+    def size(self) -> int:
+        return self.__size
+
+    def insert(self, key: str, val: Union[str, float]) -> None:
```

Comment:
Did you forget removing `-> None`?

Reply:
As Python implicitly `return None`, what I actually forgot was to add `return None` to method `remove`

Reply:
As per our recent discussion, I agree that we should remove `-> None`.